go.php

```diff
@@ -4,6 +4,8 @@
 ############## Adjust when editing docs ##################
 
 $mapping = array(
+    'admin-antivirus-configuration'     => 'admin_manual/configuration_server/antivirus_configuration.html'
+    'admin-db-conversion'     => '/admin_manual/configuration_database/db_conversion.html'
```

Comment:
Duplicate line, comma missing and leading slash missing - let me fix it and then merge.